app/views/layouts/application.html.erb

```diff
@@ -20,8 +20,12 @@
                 <div class="navbar-collapse collapse" id="navbarResponsive" style="">
                     <ul class="navbar-nav ml-auto">
                         <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#quiz">Take the Fear Quiz</a></li>
-                        <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#login">Login</a></li>
-                        <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#logout">Logout</a></li>
+                        <% if current_user.nil? %>
+                          <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#login">Login</a></li>
+                          <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#register">Register</a></li>
+                        <% else %>
+                          <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#logout">Logout</a></li>
+                        <% end %>
```

Comment:
Could we add a few more like game library, profile, home, search

Reply:
Yup, I was thinking those would be like links for a registered user's nav bar only.